### PR TITLE
Mention linting in the script/test section

### PR DESCRIPTION
**Pull request by technicalpickles**
Adding this based on the conversation in https://github.com/github/scripts-to-rule-them-all/issues/12

cc @maddox @jesperronn

### CODE REVIEW

**Comment by jesperronn** [comment]
:+1:  for mentioning it in the readme. Also, while you are at it, perhaps add a note in the actual `script/test` file?

**Comment by maddox** [comment]
@technicalpickles :+1: :shipit:

File: README.md [hunk 1]
```diff
@@ -74,12 +74,15 @@ the application is up to date and can run appropriately.
 A good pattern to support is having an optional argument that is a file path.
 This allows you to support running single tests.
 
+Linting (ie rubocop, jshint, pmd, etc) can also be considered a form of testing. These tend to run faster than tests, so put them towards the beginning of a `script/test` so it fails faster if there's a linting problem.
```
Comment (mikemcquaid) [inline README.md]: `ie` -> `i.e.`

Comment (mikemcquaid) [inline README.md]: `etc` -> `etc.` Also might be nice to monospace the command names (but feel free to ignore that). Sorry for the pedantry!

**Comment by mikemcquaid** [comment]
:+1:

**Pull request merged**